tiled/catalog/adapter.py

```diff
@@ -1226,9 +1232,20 @@ async def write_block(self, block, media_type, deserializer, entry, body):
             (await self.get_adapter()).write_block, data, block
         )
 
-    async def patch(self, shape, offset, extend, media_type, deserializer, entry, body):
+    async def patch(
+        self, shape, offset, extend, media_type, deserializer, entry, body, persist=True
+    ):
+        if extend and not persist:
```

Comment:
This can be caught in the endpoint (`router.py`) and is arguably a `400`. I would argue it is less a conflict with the current state of the server and more an incompatibility between parameters in the request. That is, we do not need to check the current state of the server to know that the request is invalid on its own terms.